torch/csrc/jit/runtime/static/ops.cpp

```diff
@@ -573,12 +573,12 @@ REGISTER_OPERATOR_FUNCTOR(aten::mul, aten_mul, [](Node* n) -> SROperator {
     const auto& in0_t = p_node->Input(0).toTensor();
     const auto& in1_t = p_node->Input(1).toTensor();
     if (p_node->Output(0).isNone()) {
-      p_node->Output(0) = at::cpu::mul(in0_t, in1_t);
+      p_node->Output(0) = at::native::mul(in0_t, in1_t);
```

Comment:
If you're curious; `at::mul()` normally means "call mul but perform dynamic dispatch", while `at::cpu::mul()` means "I know I want to call the cpu kernel for mul, skip the dynamic dispatch overhead". With this PR, `at::mul()` and `at::native::mul()` basically both mean the latter now.

Reply:
ahh makes sense